### PR TITLE
DOC: Added example for tf2zpk function

### PR DESCRIPTION
**Pull request by tamri**
#### Reference issue
DOC: Add "Examples" to docstrings #7168

#### What does this implement/fix?
Added example for tf2zpk function. 
The code can be found in scipy/signal/_filter_design.py 

#### Additional information
Once my work accepted, I will add more examples for next PR.
[skip actions] [skip cirrus]

### CODE REVIEW

File: scipy/signal/_filter_design.py [hunk 1]
```diff
@@ -1083,6 +1083,18 @@ def tf2zpk(b, a):
     transfer function coefficients must first be converted to the "positive
     powers" form before finding the poles and zeros.
 
+    Examples
+    --------
+    Find the zeroes, poles and gain of 
+    a filter with the transfer function
+        
+        H(s) = 3s^2 / (s^2 + 5s + 13)
```
Comment (j-bowhay) [inline scipy/signal/_filter_design.py:1091]: for correct rendering
```suggestion
    .. math::
    
        H(s) = \frac{3s^2}{s^2 + 5s + 13}
```

File: scipy/signal/_filter_design.py [hunk 2]
```diff
@@ -1083,6 +1083,18 @@ def tf2zpk(b, a):
...
+        
+    >>> from scipy.signal._filter_design import tf2zpk
```
Comment (j-bowhay) [inline scipy/signal/_filter_design.py:1093]: The correct public import is
```suggestion
    >>> from scipy.signal import tf2zpk
```

**Review by j-bowhay — CHANGES_REQUESTED** [review]
Thanks @tamri, some suggestions

File: scipy/signal/_filter_design.py (hunk 2 above)
Comment (tamri) [reply scipy/signal/_filter_design.py]: Thank you for the feedback

**Review by j-bowhay — APPROVED** [review]
Thanks @tamri, congratulations on your first contribution to SciPy, keep them coming!

**Comment by j-bowhay** [comment]
> Once my work accepted, I will add more examples for next PR. [skip actions] [skip cirrus]

Ps. Relating to this, please try to keep all the examples in a PR related to similar functions as this makes it easier to review

**Pull request merged**